.github/scripts/ci-test-only-normal.sh

```diff
@@ -40,6 +40,10 @@ run_all 4
 # Test heap resizing
 runbms_dacapo2006_with_heap_size fop 20 100
 
+# Test compressed oops with heap range > 4GB
+# When the heap size is larger than 4GiB, compressed oops will be shifted by 3 bits.
+runbms_dacapo2006_with_heap_size fop 20 5000
```

Comment:
I'll remove this test because https://github.com/mmtk/mmtk-openjdk/pull/292 tests with multiple plans, and GenImmix may not be the only plan affected by the compressed oops.

I'll keep the one I added to `ci-test-minimal.sh` so that it will be executed for every new commit in mmtk-core.